pytorch_lightning/metrics/functional/classification.py

```diff
@@ -630,6 +654,12 @@ def multiclass_auroc(
         >>> multiclass_auroc(pred, target, num_classes=4)
         tensor(0.6667)
     """
+    rank_zero_warn(
+        "This `multiclass_auroc` was deprecated in v1.2.0 in favor of"
+        " `pytorch_lightning.metrics.functional.auroc import auroc`."
+        " It will be removed in v1.4.0", DeprecationWarning
+    )
+
     if not torch.allclose(pred.sum(dim=1), torch.tensor(1.0)):
```

Comment:
can we internally call the new function so we do not need to maintain the code in 2 places till 1.4?

Reply:
Done :]